telegram/constants.py

```diff
@@ -1754,3 +2587,50 @@ class ForumTopicLimit(IntEnum):
     * :paramref:`~telegram.Bot.edit_general_forum_topic.name` parameter of
       :meth:`telegram.Bot.edit_general_forum_topic`
     """
+
+
+class Date(Enum):
+    """This enum contains the date literal for :class:`telegram.InaccessibleMessage`
+
+    .. versionadded:: NEXT.VERSION
+    """
```

Comment:
Having an enum with only a single member seems a bit like overkill tbh. `SUPPORTED_WEBHOOK_PORTS` & the bot api versions also still are non-enum cnostants

Reply:
It needs to be an enum, otherwise I can't literal it